src/main/resources/com/sonymobile/jenkins/plugins/lenientshutdown/ShutdownManageLink/index.jelly

```diff
@@ -0,0 +1,52 @@
+<!--
+ *  The MIT License
+ *
+ *  Copyright (c) 2014 Sony Mobile Communications Inc. All rights reserved.
```

Comment:
This was new. But I've removed this. wanted to have the same behaviour as before the clicking the link will toggle shutdown mode and not have an intermediate screen with options.